sample/Sample/CardPaymentViewController.swift

```diff
@@ -32,15 +32,26 @@ class CardPaymentViewController: UIViewController {
         return label
     }()
 
+    lazy var noticeText: UILabel = {
+        let label = UILabel()
+        label.text = "In this sample we use live production banks, so by completing the payment flow 0.01 EUR will be deducted from your account."
```

Comment:
1. Just curious, is it possible, that transaction will be more than 0.01 EUR? Like commission of some sorts and stuff. Maybe it is worth mentioning as well?
2. Will it be EUR? As fas as I can see we use account currency for payments, so if it is Polish account - Zloty would be used instead of EUR?

Reply:
Good questions. 
Ad 1. No idea :)
Ad 2. Yes, you're right but the purpose of this warning is just to make the users aware that they are using real money here. The amount and currency is second priority I guess 🤷